python/lsst/ts/ATWhiteLightSource/wlsCSC.py

```diff
@@ -236,11 +246,12 @@ async def do_emergencyPowerLightOff(self, id_data):
         """Powers the light off. This one ignores the warmup period
         that the CSC normally enforces.
         """
+        self.assert_enabled("emergencyPowerLightOff")
```

Comment:
I suggest removing the argument to `assert_enabled` *everywhere*.
it is deprecated because it is too easy to get wrong (as in this case!).

Is it necessary to call assert_enabled here? Why not keep the chiller running even if the state is, say, disabled? Naively I would expect you to test if communication is still possible. (No point trying to talk to the chiller if it has been disconnected or -- presumably -- if it is in fault).

Reply:
Why do you call `assert_enabled` here?

Naively I would expect you to keep the chiller running "if you can", which suggests that the thing to test is "do you have an open connection?" and "is the chiller operational? (as opposed to in fault)".